context/app/static/js/components/home-revision/FacetSearch/style.js

```diff
@@ -0,0 +1,33 @@
+import styled from 'styled-components';
+
+const Background = styled.div`
+  background-color: ${(props) => props.theme.palette.primary.main};
+  width: 100%;
+  padding: 20px;
+  color: #fff;
+`;
+
+const FlexForm = styled.form`
+  display: flex;
+  align-items: center;
+  flex-wrap: wrap;
+`;
+
+const StyledLabel = styled.label`
+  margin-right: ${(props) => props.theme.spacing(5)}px;
+`;
+
+const inputPadding = 8;
+
+const StyledInput = styled.input`
+  flex-grow: 1;
+  appearance: none !important;
+  outline: none;
+  border-radius: 8px;
+  border: 0;
+  padding: ${inputPadding}px;
+  font-size: 16px;
```

Comment:
I'm not clear on when we use `rem` and when we use `px`.

Reply:
This was an oversight on my part. `rem` should be used for `font-size`.